src/main/java/History.java

```diff
@@ -0,0 +1,51 @@
+package com.google.sps.data;
+import com.google.sps.data.VisualizationData;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import java.util.ArrayList;
+import java.util.List;
```

Comment:
Doesn't look like you're using this so could remove this line

Reply:
fixed